libraries/AP_Motors/AP_MotorsCoax.cpp

```diff
@@ -188,11 +188,22 @@ void AP_MotorsCoax::output_armed_stabilizing()
         limit.yaw = true;
     }
 
-    _thrust_yt_ccw = thrust_out + 0.5f * yaw_thrust;
-    _thrust_yt_cw = thrust_out - 0.5f * yaw_thrust;
+    // calculate the throttle setting for the lift fan
+    float thrust_yt_ccw = thrust_out + 0.5 * yaw_thrust;
+    float thrust_yt_cw = thrust_out - 0.5 * yaw_thrust;
+
+    // Apply slew limit to thrust output
+    if (!is_zero(_slew_time)) {
+        float thrust_rpyt_out_delta_max = 1.0 / (_slew_time * _loop_rate);
```

Comment:
can we use a multiply here?

Reply:
I don't know how we can use a multiply there.